src/nakamoto-signer-developer-guide.md

```diff
@@ -0,0 +1,196 @@
+# What is this
+
+Draft documentation for sBTC Signer specifically for the Nakamoto [release](https://stacks-network.github.io/sbtc-docs/sbtc-roadmap.html)
+
+The readers of this are assumed to be anyone who is looking to run a sBTC Signer with minimal custom configuration.
+
+# Disclaimer
+
+Development is on going and changes to the Signer configuration should be expected. This document will be updated every sprint to ensure its accuracy.
+
+## Prerequisites
+
+Rust - [To install](https://www.rust-lang.org/tools/install).
+
+Accessible Stacks node - A Stacks node running the stackerDB instance which is used for signer communication and transaction monitoring and broadcasting.
+
+Accessible Bitcoin node - A Bitcoin node used for transaction monitoring and broadcasting.
+
+A Stacks Address - Identify your address as a Signer
```

Comment:
Might be more accurate to say "A Stacks Private Key - For signing sBTC transactions and validating blocks."

Reply:
fixed 97695ec9ce37137f9836f3e62bd22c7701fb51de